### DISCUSSION TITLE
Free ChatGTP?

### DISCUSSION THREAD

**Discussion by zkjv** [community/community#132864] [Programming Help] [answered]
### Body

i have an idea in mind, to make one but with jailbreak.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hriheirv** [comment, marked as answer]
damn, yes pls

**Comment by yaskaawad348** [comment]
- Hack [yaska](url) 

> `****`

1. 

> 

> ### [**_[](url)_**](yaska money)

> ``Hack
Fuck 

> [money](url)

- [ ] 

- [ ] 1. > awad[](url)
